libcudacxx/test/utils/nvidia/nvrtc/nvrtcc.cpp

```diff
@@ -163,6 +163,20 @@ ArgPair argHandlers[] = {
      nvrtcArguments.emplace_back("-G");
      return NORMAL;
    }},
+  {// Matches --device-int128/-device-int128
+   std::regex("^(?:--device-int128|-device-int128)$"),
+   [](const std::smatch&) {
+     nvrtcArguments.emplace_back("-device-int128");
+     return NORMAL;
+   }},
```

Comment:
This needs to be guarded by SM100

 nvrtc: error: option --device-float128 is not supportedon architectures earlier than sm_100

Reply:
We should do both, I can Imagine people using this code as a blueprint for driving nvrtc so we should be safe on this side

Reply:
I don't think we should have to worry about doing both?

Doesn't NVRTC fail if we pass flags that aren't supported by the architecture?